src/GNDStk/v1.9/containers/XYs1d.hpp

```diff
@@ -96,39 +96,51 @@ class XYs1d : public Component<XYs1d> {
    // ------------------------
 
    // index
-   const auto &index() const
+   const std::optional<Integer32> &
+   index() const
     { return content.index; }
-   auto &index()
+   std::optional<Integer32> &
+   index()
     { return content.index; }
 
    // interpolation
-   const auto &interpolation() const
+   const enums::Interpolation
+   interpolation() const
     { return content.interpolation.value(); }
-   auto &interpolation()
+   enums::Interpolation
+   interpolation()
     { return content.interpolation.value(); }
```

Comment:
I was wrong, here's another one where you're missing the reference.

Reply:
Again, this is because `interpolation` is a `Defaulted`. We need to give Python the correct understanding of `Defaulted`, and then can get rid of the `.value()` and make everything consistent.